miqcli/constants.py

```diff
@@ -13,22 +13,46 @@
 #    You should have received a copy of the GNU General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-
+"""
+miqcli.constants values will differ from installation to installation.
+"""
 import os
 
-VERSION = '0.0.0'
+from pbr.packaging import get_version
+
+
+#: Installed version of miqcli
+VERSION = get_version('miqcli')
```

Comment:
I just noticed that if we do this, we should swap VERSION and PACKAGE, so VERSION is `get_version(PACKAGE)`